2023-04-20 16:39:03,437 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Configure stats pid to 35392
2023-04-20 16:39:03,438 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Loading settings from C:\Users\shanthod\.config\wandb\settings
2023-04-20 16:39:03,438 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Loading settings from C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\settings
2023-04-20 16:39:03,438 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-04-20 16:39:03,438 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-20 16:39:03,438 INFO    MainThread:35392 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230420_163903-074c7x04\logs\debug.log
2023-04-20 16:39:03,438 INFO    MainThread:35392 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230420_163903-074c7x04\logs\debug-internal.log
2023-04-20 16:39:03,438 INFO    MainThread:35392 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001C1A9FEAF10>
2023-04-20 16:39:03,439 INFO    MainThread:35392 [wandb_init.py:init():532] calling init triggers
2023-04-20 16:39:03,439 INFO    MainThread:35392 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-04-20 16:39:03,439 INFO    MainThread:35392 [wandb_init.py:init():588] starting backend
2023-04-20 16:39:03,439 INFO    MainThread:35392 [wandb_init.py:init():592] setting up manager
2023-04-20 16:39:03,442 INFO    MainThread:35392 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-04-20 16:39:03,451 INFO    MainThread:35392 [wandb_init.py:init():599] backend started and connected
2023-04-20 16:39:03,461 INFO    MainThread:35392 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-04-20 16:39:03,463 INFO    MainThread:35392 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-04-20 16:39:03,463 INFO    MainThread:35392 [wandb_init.py:init():687] updated telemetry
2023-04-20 16:39:03,470 INFO    MainThread:35392 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-04-20 16:39:04,118 INFO    MainThread:35392 [wandb_run.py:_on_init():2134] communicating current version
2023-04-20 16:39:04,169 INFO    MainThread:35392 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.15.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-04-20 16:39:04,169 INFO    MainThread:35392 [wandb_init.py:init():775] starting run threads in backend
2023-04-20 16:39:04,307 INFO    MainThread:35392 [wandb_run.py:_console_start():2114] atexit reg
2023-04-20 16:39:04,307 INFO    MainThread:35392 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-04-20 16:39:04,308 INFO    MainThread:35392 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-04-20 16:39:04,308 INFO    MainThread:35392 [wandb_run.py:_redirect():2059] Redirects installed.
2023-04-20 16:39:04,308 INFO    MainThread:35392 [wandb_init.py:init():817] run started, returning control to user process
2023-04-20 16:39:04,310 INFO    MainThread:35392 [wandb_run.py:_config_callback():1250] config_cb None None {'alias': '', 'experiment_id': '', 'run_tags': [], 'run_notes': '', 'seed': 0, 'device': 'cpu', 'dataset': 'Jet_Engine', 'reciprocal': True, 'model': 'ComplEx', 'rank': 1000, 'rank_r': 100, 'init': 0.001, 'dropout': 0, 'lmbda': 0.05, 'world': 'LCWA', 'num_neg': 0, 'score_rel': True, 'score_lhs': False, 'score_rhs': True, 'w_rel': 4.0, 'w_lhs': 1, 'regularizer': 'N3', 'optimizer': 'Adagrad', 'max_epochs': 100, 'valid': 3, 'batch_size': 1000, 'learning_rate': 0.1, 'decay1': 0.9, 'decay2': 0.999, 'cache_eval': None, 'model_cache_path': None, 'size': [486, 124, 486], 'loss': 'CrossEntropyLoss()'}
2023-04-20 16:39:04,311 INFO    MainThread:35392 [wandb_watch.py:watch():51] Watching
2023-04-20 16:40:56,509 INFO    MainThread:35392 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-04-20 16:40:56,510 INFO    MainThread:35392 [wandb_init.py:_pause_backend():403] pausing backend
2023-04-20 16:41:04,966 INFO    MainThread:35392 [wandb_init.py:_resume_backend():408] resuming backend
2023-04-20 16:41:05,024 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Configure stats pid to 35392
2023-04-20 16:41:05,024 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Loading settings from C:\Users\shanthod\.config\wandb\settings
2023-04-20 16:41:05,024 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Loading settings from C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\settings
2023-04-20 16:41:05,024 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-04-20 16:41:05,024 INFO    MainThread:35392 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-20 16:41:05,025 INFO    MainThread:35392 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230420_164105-u5gcnv1k\logs\debug.log
2023-04-20 16:41:05,025 INFO    MainThread:35392 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\shanthod\Downloads\ssl-relation-prediction-main\ssl-relation-prediction-main\src\wandb\run-20230420_164105-u5gcnv1k\logs\debug-internal.log
2023-04-20 16:41:05,025 INFO    MainThread:35392 [wandb_init.py:init():532] calling init triggers
2023-04-20 16:41:05,025 INFO    MainThread:35392 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-04-20 16:41:05,025 INFO    MainThread:35392 [wandb_init.py:init():554] re-initializing run, found existing run on stack: 074c7x04
2023-04-20 16:41:05,027 INFO    MainThread:35392 [wandb_run.py:_finish():1854] finishing run shan_proj/ssl-relation-prediction/074c7x04
2023-04-20 16:41:05,027 INFO    MainThread:35392 [jupyter.py:save_history():449] not saving jupyter history
2023-04-20 16:41:05,027 INFO    MainThread:35392 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-04-20 16:41:05,027 INFO    MainThread:35392 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-04-20 16:41:05,027 INFO    MainThread:35392 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-04-20 16:41:05,027 INFO    MainThread:35392 [wandb_run.py:_restore():2066] restore
2023-04-20 16:41:05,028 INFO    MainThread:35392 [wandb_run.py:_restore():2072] restore done
2023-04-20 16:41:12,731 INFO    MainThread:35392 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-04-20 16:41:12,732 INFO    MainThread:35392 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-04-20 16:41:12,742 INFO    MainThread:35392 [wandb_run.py:_footer_sync_info():3398] logging synced files
